README.md

```diff
@@ -228,6 +229,55 @@ jobs:
 
 > **_NOTE:_** Building Storybook locally makes it simple to test Storybooks that could be available remotely, but are under authentication layers. If you also deploy your Storybooks somewhere (e.g. Chromatic, Vercel, etc.), the Storybook URL can still be useful with the test-runner. You can pass it to the `REFERENCE_URL` environment variable when running the test-storybook command, and if a story fails, the test-runner will provide a helpful message with the link to the story in your published Storybook instead.
 
+## Experimental test hook API
+
+The test runner renders a story and executes its [play function](https://storybook.js.org/docs/react/writing-stories/play-function) if one exists. However, there are certain behaviors that are not possible to achieve via the play function, which executes in the browser. For example, if you want the test runner to take visual snapshots for you, this is something that is possible via Playwright/Jest, but must be executed in Node.
+
+To enable use cases like visual or DOM snapshots, the test runner exports test hooks that can be overridden globally. These hooks give you access to the test lifecycle before and after the story is rendered.
+
+Consider the following pseudocode:
+
+```js
+it('component--widget', async () => {
```

Comment:
Is this pseudo-code, written per-test, necessary _along with_ the more global code in the next snippet?

Reply:
Thanks I'll rewrite this part to clarify

Reply:
@kylegach updated to (hopefully) make it more clear

Reply:
Yes, that makes it _much_ more clear! I wasn't grokking the "automatically generated" nature of it, before.